### PR TITLE
Copy original APIs

### PR DESCRIPTION
**Pull request by kron4eg**
This is a copy of original kubermatic CRDs that will be changed further in the follow-up PRs, including changing Group (from GVK)

xref #7866

```release-note
NONE
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -123,7 +123,6 @@ endif
 lint:
 	golangci-lint run \
 		--verbose \
-		--build-tags "$(KUBERMATIC_EDITION)" \
```
Comment (xrstf) [inline Makefile:126]: This would make it impossible to lint CE or EE.

**Review by xrstf — COMMENTED** [review]
Commentatio!

File: Makefile (hunk 1 above)
Comment (kron4eg) [reply Makefile:126]: Turns out I've forgot to add ce/ee tags :D

Comment (kron4eg) [reply Makefile:126]: Fixed

**Review by xrstf — APPROVED** [review]
/approve

**Pull request merged**